modules/administration/pages/tshoot-sync.adoc

```diff
@@ -65,3 +65,14 @@ spacewalk-repo-sync --channel <channelname> -Y
 ----
 +
 This option verifies the repository data before the synchronization, rather than relying on locally cached checksums.
+
+Connection Timeout::
+If the download times out with the following error:
++
+----
+28, 'Operation too slow. Less than 1000 bytes/sec transferred the last 300 seconds
+----
++
+You can resolve this error by specifying [literal]``reposync_timeout`` and [literal]``reposync_minrate`` configuration
+values in [path]``/etc/rhn/rhn.conf``. When less than [literal]``reposync_minrate`` bytes per second (default:
+[literal]``1000``) are transfered for a period of [literal]``reposync_timeout`` seconds (default: [literal]``300``) the download is aborted.
```

Comment:
```suggestion
You can resolve this error by specifying [literal]``reposync_timeout`` and [literal]``reposync_minrate`` configuration values in [path]``/etc/rhn/rhn.conf``. 
By default, when less than 1000 bytes per second are transferred in 300 secs, the download is aborted.
You can adjust the number of bytes per second with [literal]``reposync_minrate``, and the number of seconds to wait with [literal]``reposync_timeout``.
```

Just trying to make that second sentence a bit clearer, and also adjust the newline syntax to match the rest of the doc.